pkg/volume/quobyte/quobyte_test.go

```diff
@@ -98,8 +98,9 @@ func doTestPlugin(t *testing.T, spec *volume.Spec) {
 		t.Error("Got a nil Mounter")
 	}
 
-	if volumePath != fmt.Sprintf("%s/plugins/kubernetes.io~quobyte/root#root@vol", tmpDir) {
-		t.Errorf("Got unexpected path: %s expected: %s", volumePath, fmt.Sprintf("%s/plugins/kubernetes.io~quobyte/root#root@vol", tmpDir))
+	expectedPath := fmt.Sprintf("%s/plugins/kubernetes.io~quobyte/mnt/root#root@vol", tmpDir)
```

Comment:
You should use `qbMountPointName` instead of directly add `mnt`(e.g. if you want to change the path later)

Reply:
Absolutely, done.